products/gstreamer.md

```diff
@@ -0,0 +1,81 @@
+---
+title: GStreamer
+category: framework
+iconSlug: gstreamer
+permalink: /gstreamer
+versionCommand: gst-inspect-1.0 --version
+releasePolicyLink: https://gstreamer.freedesktop.org/
+changelogTemplate: https://gstreamer.freedesktop.org/data/pkg/src/1.11.90/{{"__RELEASE_CYCLE__"}}
+releaseLabel: "__RELEASE_CYCLE__"
+activeSupportColumn: false
+releaseDateColumn: true
+eolColumn: Supported
+
+auto:
+-   git: https://gitlab.freedesktop.org/gstreamer/gstreamer.git
+    regex: '^(?<major>[1-9]\d*)\.(?<minor>0|[1-9]\d*)\.?(?<patch>0|[1-9]\d*)?$'
+
+releases:
+-   releaseCycle: "1.22"
+    releaseDate: 2023-01-23
+    eol: false
+    latest: "1.22.2"
+    latestReleaseDate: 2023-04-11
+
+-   releaseCycle: "1.20"
+    releaseDate: 2022-02-03
```

Comment:
Release 1.20 and lower are missing the `eol` field. If the exact date is not known, you can use `true` or `false`.